http_cache.rst

```diff
@@ -77,23 +77,29 @@ but is a great way to start.
 
     For details on setting up Varnish, see :doc:`/http_cache/varnish`.
 
-Enabling the proxy is easy: each application comes with a caching kernel (``AppCache``)
-that wraps the default one (``AppKernel``). The caching Kernel *is* the reverse
-proxy.
+To enable the proxy, first create a caching kernel::
 
-To enable caching, modify the code of your front controller. You can also make these
-changes to ``index.php`` to add caching to the ``dev`` environment::
+    // src/CacheKernel.php
+    <?php
+
+    use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+    class CacheKernel extends HttpCache
+    {
+    }
+
+Modify the code of your front controller to wrap the default kernel into the
+caching kernel::
 
     // public/index.php
```

Comment:
Let's change the `::` to just `:` and add (on the next line) `.. code-block:: diff`.

Then, you can add + and - to emphasize the changes:

```
.. code-block:: diff

    // some line from before
    - // line to go away
    + // line to add
```

Reply:
That's great advice, thanks :) I could check that everything is fine by compiling the doc with Sphinx.